src/art/utils/trajectory_logging.py

```diff
@@ -25,21 +33,32 @@ def trajectory_group_to_dict(trajectory_group: TrajectoryGroup) -> dict[str, Any
         "metadata": trajectory_group.metadata,
     }
 
+
 def trajectory_to_dict(trajectory: Trajectory) -> dict[str, Any]:
-    messages_and_choices = [message_or_choice_to_dict(message_or_choice) for message_or_choice in trajectory.messages_and_choices]
+    messages_and_choices = [
+        message_or_choice_to_dict(message_or_choice)
+        for message_or_choice in trajectory.messages_and_choices
+    ]
 
     return {
         "reward": trajectory.reward,
         "metrics": trajectory.metrics,
         "metadata": trajectory.metadata,
         "messages_and_choices": messages_and_choices,
+        "tools": trajectory.tools,
```

Comment:
These two lines are the only real change.

Reply:
Separate question: should this be combined with the `for_logging` function in trajectories.py?